config/templates/stack-template.json

```diff
@@ -123,11 +130,20 @@
           }
         ],
         "VPCZoneIdentifier": [
+          {{if $.Controller.TopologyPrivate}}
+          {{range $index, $controllerSubnet := .Controller.PrivateSubnets}}
+          {{with $controllerSubnetLogicalName := printf "Controller%s" $controllerSubnet.LogicalName}}
+          {{if gt $index 0}},{{end}}
+          {
+            "Ref": "{{$controllerSubnetLogicalName}}"
+          }
+          {{end}}
+          {{end}}
+          {{else}}
           {{range $index, $subnet := .Subnets}}
```

Comment:
Of course, we can. Look at this [comment](https://github.com/coreos/kube-aws/pull/169#discussion_r95475310)
 I though it was a good idea to wait for @mumoshu approval before doing more work